### PR TITLE
16 search result component

### PR DESCRIPTION
**Pull request by SajedaIsmail**
- Change the favicon to the sitspot logo
- Create search result list and style it
Relates #16

### CODE REVIEW

File: client/src/components/HeaderComponent/index.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import logo from '../../assets/images/logo.svg';
+import './style.css';
+
+export default class Header extends React.Component {
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/index.js]:
```suggestion
export default class Header extends Component {
```

File: client/src/components/HeaderComponent/index.js [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+          <header>
+            <img alt="logo" src={logo} />
+            <button
+              className="burger"
+              type="button"
+              onClick={this.onClickHandler}
+            >
+              <div />
+              <div />
+              <div />
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/index.js]: why empty divs ?

File: client/src/components/HeaderComponent/style.css [hunk 3]
```diff
@@ -0,0 +1,19 @@
+button > div {
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/style.css]: use classess to style elements

File: client/src/components/HeaderComponent/style.css [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+  width: 5rem;
+  height:0.5rem;
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/style.css]:
```suggestion
  height: 0.5rem;
```

File: client/src/components/HeaderComponent/style.css [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+  background-color: white;
+  margin: 1rem 0;
+}
+.burger{
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/style.css]:
```suggestion
.burger {
```

File: client/src/components/HeaderComponent/style.css [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+  background-color: transparent;
+  margin-inline-end: 2%;
+  border: 0ch;
+}
+ header{
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/style.css]:
```suggestion
 header {
```
use class Names to style elements

File: client/src/components/HeaderComponent/style.css [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+   width: 100%;
+   height: 10%;
+   background-color: transparent;
+   display: flex;
+   justify-content: space-between;
+  
+ }
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/style.css]:
```suggestion
 }
 
```
end of file line

File: client/src/components/SearchResultComponent/index.js [hunk 8]
```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import propTypes from 'prop-types';
+import image from '../../assets/images/placeimg.png';
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/index.js]:
```suggestion

import image from '../../assets/images/placeimg.png';
```

File: client/src/components/SearchResultComponent/style.css [hunk 9]
```diff
@@ -0,0 +1,37 @@
+.list{
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
.list {
```

File: client/src/components/SearchResultComponent/style.css [hunk 10]
```diff
@@ -0,0 +1,37 @@
...
+  width: 100%;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content:flex-start;
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
  justify-content: flex-start;
```

File: client/src/components/SearchResultComponent/style.css [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+  align-items: flex-start;
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]: flex start is the default value, you don't need to use it

File: client/src/components/SearchResultComponent/style.css [hunk 12]
```diff
@@ -0,0 +1,37 @@
...
+  align-content: flex-start;
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]: flex start is the default value, you don't need to use it

File: client/src/components/SearchResultComponent/style.css [hunk 13]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+.list_item{
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
.list_item {
```

File: client/src/components/SearchResultComponent/style.css [hunk 14]
```diff
@@ -0,0 +1,37 @@
...
+  flex-wrap: nowrap;
+  justify-content: flex-start;
+  align-items: flex-start;
+  align-content: flex-start;
+  margin: 3%;
+
+}
+
+
+.list_item_img{
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
.list_item_img {
```

File: client/src/components/SearchResultComponent/style.css [hunk 15]
```diff
@@ -0,0 +1,37 @@
...
+  width: 100%;
+  height: 10rem;
+  padding: 1%;
+}
+ 
+
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
```

File: client/src/components/SearchResultComponent/style.css [hunk 16]
```diff
@@ -0,0 +1,37 @@
...
+li > p {
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]: use classes to style components

File: client/src/components/SearchResultComponent/style.css [hunk 17]
```diff
@@ -0,0 +1,37 @@
...
+color: #FFFFFF;
+font-family: "Raleway",sans-serif;
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
font-family: "Raleway", sans-serif;
```

File: client/src/App/index.js [hunk 18]
```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import './style.css';
 
 const App = () => {
-  return <div className="App"> </div>;
+  return <div className="App" />;
 };
 
 export default App;
```
Comment (mossa-Sammer) [inline client/src/App/index.js]: why this file is here?

File: client/src/components/HeaderComponent/style.css [hunk 19]
```diff
@@ -0,0 +1,18 @@
...
+  height:0.5rem;
+  background-color: white;
+  margin: 1rem 0;
+}
+.burger{
+  background-color: transparent;
+  margin-inline-end: 2%;
+  border: 0ch;
+}
+ .nav_header{
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/style.css]:
```suggestion
 .nav_header {
```

File: client/src/components/HeaderComponent/style.css [hunk 20]
```diff
@@ -0,0 +1,18 @@
...
+   width: 100%;
+   height: 10%;
+   background-color: transparent;
+   display: flex;
+   justify-content: space-between;
+ }
```
Comment (mossa-Sammer) [inline client/src/components/HeaderComponent/style.css]:
```suggestion
 }
 
```

File: client/src/components/SearchResultComponent/style.css [hunk 21]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.list_item{
+  display: flex;
+  flex-direction: column;
+  flex-wrap: nowrap;
+  align-items: flex-start;
+  align-content: flex-start;
+  margin: 3%;
+
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
```

File: client/src/components/SearchResultComponent/style.css [hunk 22]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+.list_item_img{
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
.list_item_img {
```

File: client/src/components/SearchResultComponent/style.css [hunk 23]
```diff
@@ -0,0 +1,33 @@
...
+  width: 100%;
+  height: 10rem;
+  padding: 1%;
+}
+ 
+.element_text{
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
.element_text {
```

File: client/src/components/SearchResultComponent/style.css [hunk 24]
```diff
@@ -0,0 +1,33 @@
...
+color: rgb(48, 46, 46);
+font-family: "Raleway", sans-serif;
+font-size: 1.85rem;
+font-weight: 650;
+padding: 1%;
+text-align: start;
+}
```
Comment (mossa-Sammer) [inline client/src/components/SearchResultComponent/style.css]:
```suggestion
}

```

**Comment by Amoodaa** [comment]
use a prettier to format css, right click on the editor and then format document, it may ask u to configure a default formatter, use prettier, so we have some consistency

**Pull request closed without merging**